mathesar/imports/csv.py

```diff
@@ -72,14 +72,16 @@ def get_sv_dialect(file):
 def get_sv_reader(file, header, dialect=None):
     file = TextIOWrapper(file, encoding="utf-8-sig")
     if dialect:
-        reader = csv.DictReader(file, dialect=dialect)
+        reader = csv.DictReader(file, dialect=dialect, quoting=csv.QUOTE_ALL, quotechar='"', delimiter=',')
     else:
-        reader = csv.DictReader(file)
+        reader = csv.DictReader(file, quoting=csv.QUOTE_ALL, quotechar='"', delimiter=',')
     if not header:
         reader.fieldnames = [
             f"column_{i}" for i in range(len(reader.fieldnames))
         ]
         file.seek(0)
+    else:
```

Comment:
Do we need to still do this if we're handling the quotes at the SQL level?

Reply:
I don't think we need to keep this part. So I'll remove it.